uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Page/Dashboard.php

```diff
@@ -6,6 +6,23 @@
 class CRM_HRLeaveAndAbsences_Page_Dashboard extends CRM_Core_Page {
   public function run() {
     CRM_Utils_System::setTitle(ts('Dashboard'));
+
+    CRM_Core_Resources::singleton()
+      ->addStyleFile('uk.co.compucorp.civicrm.hrleaveandabsences', 'css/leaveandabsence.css')
+      ->addScriptFile('uk.co.compucorp.civicrm.hrleaveandabsences',
+        CRM_Core_Config::singleton()->debug ? 'js/angular/src/admin-dashboard.js' : 'js/angular/dist/admin-dashboard.min.js', 1010)
+      ->addSettingsFactory(function () {
+        return array(
```

Comment:
@davialexandre I actually confused myself, it becomes available once you actually put it in CRM.vars.leaveAndAbsences via addVars. So your suggestion is correct.